modules/devguide/examples/nodejs/cloud.js

```diff
@@ -1,61 +1,46 @@
-var couchbase = require('couchbase');
+var couchbase = require('couchbase')
 
-// Update this to your cluster
-
-const endpoint = 'cb.f334ae21-0eb9-4ebb-83b7-9a6f005745ed.dp.cloud.couchbase.com'
+const clusterConnStr =
+  'couchbases://cb.abcdefab-cdef-abcd-efab-cdefabcdef.dp.cloud.couchbase.com'
+const cloudRootCertificate = '/etc/x509-cert/SSLCA/clientdir/trust.pem'
 const username = 'user'
 const password = 'password'
 const bucketName = 'couchbasecloudbucket'
 
-// User Input ends here.
-
-// Initialize the Connection
-var cluster = new couchbase.Cluster('couchbases://' +endpoint+'?ssl=no_verify&console_log_level=5', {username: username, password: password});
-var bucket = cluster.bucket(bucketName);
-var collection = bucket.defaultCollection();
+async function go() {
+  const cluster = await couchbase.connect(clusterConnStr, {
+    username: username,
+    password: password,
+    trustStorePath: cloudRootCertificate,
+  })
 
-function start(){
-  console.log('start');
-  return new Promise( (resolve, reject) => { resolve(); });
-}
+  const bucket = cluster.bucket(bucketName)
+  const collection = bucket.defaultCollection()
 
-async function run(){
-    // Create a N1QL Primary Index (but ignore if it exists)
-  try {
-    await cluster.queryIndexes().createPrimaryIndex(bucketName, {ignoreExists: true});
-  } catch (e) {
-  }
+  // Create a N1QL Primary Index (but ignore if it exists)
+  await cluster
+    .queryIndexes()
+    .createPrimaryIndex(bucketName, { ignoreExists: true })
 
   // Create and store a document
-  try {
-    await collection.upsert('user:king_arthur', {
-        'name': 'Arthur', 'email': 'kingarthur@couchbase.com', 'interests': ['Holy Grail', 'African Swallows']
-    });
-  } catch (e) {
-    throw(e);
-  }
+  await collection.upsert('user:king_arthur', {
+    name: 'Arthur',
+    email: 'kingarthur@couchbase.com',
+    interests: ['Holy Grail', 'African Swallows'],
+  })
 
-    // Load the Document and print it
-    // Prints Content and Metadata of the stored Document
-  try {
-    let getResult = await collection .get('user:king_arthur');
-    console.log('Got: ');
-    console.log(getResult);
-  } catch (e) {
-    console.log(e);
-    throw(e);
-  }
+  // Load the Document and print it
+  // Prints Content and Metadata of the stored Document
+  let getResult = await collection.get('user:king_arthur')
+  console.log('got: ', getResult)
 
   // Perform a N1QL Query
-  const options = { parameters: ['African Swallows'] };
-  try {
```

Comment:
You've removed the try/catch wrappers from all of these, is that preferred style in general?

(e.g. thinking of the other Node SDK examples. e.g.  for Collections examples, I added them, to give some kind of documentation of which errors were expected. But I can see that omitting looks cleaner.

Will any error arising be surfaced naturally? Asking as, oddly, with C#, I've had situation where async errors simply got swallowed by the framework if not explicitly handled...)